contents/docs/getting-started/next-steps.mdx

```diff
@@ -0,0 +1,107 @@
+
+Warning: watching replays is addictive, but incredibly useful.
+
+## 5. Start sending PostHog data to other tools
+
+As data arrives in PostHog, you can export it _immediately_ to other tools using [realtime destinations](/docs/cdp/destinations/). 
+
+You can use this for things like:
+
+- Conversion tracking
+- Monitoring key events with your chat platform
+- Enriching customer data in your CRM
+- Triggering automations
+
+We have dozens of pre-configured destinations, including [Slack](/docs/cdp/destinations/slack), [Discord](/docs/cdp/destinations/discord), [Hubspot](/docs/cdp/destinations/hubspot), [Intercom](/docs/cdp/destinations/hubspot), and a [webhook destination](/docs/cdp/destinations/webhook) for sending events to any HTTP endpoint.
+
+Head to the [data pipelines](/docs/cdp) docs for more on how you can use PostHog as your customer data platform.
+
+You can also configure [batch exports](/docs/cdp/batch-exports/s3) to a data warehouse.
+
+## 6. Import revenue and customer data into our data warehouse
+
+When you choose PostHog, you also get a built-in data warehouse.
+
+You can then [link your most valuable data](/docs/data-warehouse/join) from sources like your CRM, payment processor, or database, combine it with the event data already in PostHog, and query across all of it.
+
+We currently offer fully-managed import integrations for the likes of [Stripe](/docs/cdp/sources/stripe), [Hubspot](/docs/cdp/sources/hubspot), and [Salesforce](/docs/cdp/sources/salesforce).
+
+We also support important data from most popular data warehouses and databases, such as [Postgres](/docs/cdp/sources/postgres), [BigQuery](/docs/cdp/sources/bigquery), and [Amazon S3](/docs/cdp/sources/s3).
+
+See the full [list of PostHog data sources](/docs/cdp/sources) for more. 
+
+## 7. Figure out what to track in PostHog
+
+PostHog has [autocapture](/docs/product-analytics/autocapture), so you're already collecting useful data from the moment you install our snippet.
+
+But you'll still need to setup some custom events to get the most value from PostHog.
+
+If you have no idea where to start, we recommend reading our guide to [Winning with PostHog](/docs/new-to-posthog/getting-hogpilled). It will guide you through:
+
+- [Creating a tracking plan](/docs/new-to-posthog/getting-hogpilled) by defining your North Star metric.
+- [How to track and measure activation](/docs/new-to-posthog/activation) – aka one of the most important metrics ever.
+- [How to think about and track retention](/docs/new-to-posthog/retention) of users and customers.
+- [Capturing revenue in PostHog](/docs/new-to-posthog/revenue), so you can monitor the health of your whole business.
+
+![a complete dashboard](https://res.cloudinary.com/dmukukwp6/image/upload/hogflix_c681b6891d.png)
+
+## 8. Start testing in production
```

Comment:
Could we make this a point about both feature flags and experiments? The title has a sort of double meaning. Maybe this is cramping too many points in one section.